src/leap/mail/smtp/tests/test_gateway.py

```diff
@@ -23,6 +23,7 @@
 import re
```

Comment:
All the smtp tests fail, but this might be my fault as I think all the modifications here are done by me.